packages/ui-extensions/src/surfaces/admin/components/AdminBlock.d.ts

```diff
@@ -39,6 +39,7 @@ export interface ClickOptions {
  */
 declare const BaseClass: typeof globalThis.HTMLElement;
 declare abstract class PreactCustomElement extends BaseClass {
+  #private;
```

Comment:
Looks like this is added to every individual [componentName].d.ts file
Is this intended? @billfienberg